src/graphql/lookupTablesData.ts

```diff
@@ -0,0 +1,19 @@
+import { GraphQLScalarType } from 'graphql';
+import { ObjectType, Field } from 'type-graphql';
+
+const lookupOptionArrayType = new GraphQLScalarType({ name: 'lookupOptionArrayType' });
+
+export class LookupOption {
+  public value: string;
+  public translationCode?: string;
+  public properties: Record<string, unknown>;
+}
+
+@ObjectType()
+export class LookupTablesData {
+  @Field(() => lookupOptionArrayType)
```

Comment:
Change the scalarType to: lookupOptionType, so that it refers only to the objects inside the array.

example:
@Field(() => [lookupOptionType])
  public countryList: LookupOption[];